### ISSUE TITLE
Table cells not responding to key presses and exhibiting wrong behaviour

### ISSUE THREAD

**Issue by LeeMbarrett** [jgraph/drawio#2705]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
There are two related issues concerning Tables.

1. When I select a table cell and start typing, Draw IO seems to be assuming that I'm not typing anything. The cursor does not appear and whatever I type does not get rendered in the cell. I have tried double-clicking into the cells (which should be unnecessary) and the cursor still does not appear.
2. For those cells which do have text in them I have to double click just to get into the cell but double-clicking does not guarantee that the cell will get focus and show a cursor (sometime I have to click many times). The entire text gets highlighted (which is the wrong behaviour and should only position the cursor next to (or between) characters - not select everything). I should be able to press the delete key or just start typing to overwrite the text but draw IO ignores my key presses until I select un-select the highlighted text.

**To Reproduce**
1. Create a "table with Title"
2. Click on one cell of that table
3. Start typing (nothing happens)
4. Double click into the same cell and start typing (again, nothing happens)
5. Frantically click many times into the same cell just to see the cursor ... again, nothing happens.

**Expected behavior**
I expect tables to work just as simply and cleanly as a spreadsheet following these rules as observed in Excel:
1. If a cell is empty and I single-click that cell and start typing then I expect to see the cursor and any text I type.
2. If the cell already has content and I single-click that cell then I expect the text to be overwritten with new text.
3. If the cell already has content and I double-click that cell I expect the cursor position to be between (or next to) the characters where the cursor is.

**Screenshots**
I have single and double-clicked a cell but the cell is unresponsive to my key-presses (no cursor, no response from keyboard)
![image](https://user-images.githubusercontent.com/55362361/159453759-5582d6a9-9772-4b4f-94c8-6434aaaf0766.png)

I have double-clicked into the cell, the entire text is selected (wrong behaviour). I start typing and the text is not overwritten (nothing happens). Only if I click on the text one more time will I see the cursor after which I can then press CTRL-A and continue typing or press del.
![image](https://user-images.githubusercontent.com/55362361/159454259-79f05e95-b11b-44e9-9685-d74bdb933970.png)

**draw.io version (In the Help->About menu of the draw.io editor):**
- draw.io version 17.1.3

**Desktop (please complete the following information):**
 - OS: Windows 10 (Version 10.0.19044 Build 19044)
 - Browser: Google Chrome (Version 99.0.4844.74 (Official Build) (64-bit))

**Smartphone (please complete the following information):**
- n/a

**Additional context**

**Comment by mararad** [comment]
Could you test this in other browsers and Incognito mode? Also hard reload it and you'll see 17.1.5 version (re-test it please)

I tested in Chrome Version 99.0.4844.74 (Official Build) (64-bit), digrams.net 17.1.5 and I couldn't reproduce the issues.

"double-clicked a cell but the cell is unresponsive to my key-presses (no cursor, no response from keyboard)" - double click shows cursor at my place

"I have double-clicked into the cell, the entire text is selected (wrong behaviour). I start typing and the text is not overwritten (nothing happens)"  - text is overwritten 

If nothing helps, please attach the file

**Comment by LeeMbarrett** [comment]
Thanks for the reply. I retried a couple of days ago (sorry for not replying same day).

I believe I now know the cause of the problem which you might be able to replicate as follows:
1. Add the 'props' plugin
2. Create any object that holds text - I've tested on Tables and Boxes.
3. Try to edit the cell or box

I understand that plugins are not part of the core functionality and as such are unsupported but are the plugins produced by the DrawIO developers or are they 3rd party?
